locales-pending/landing-all.ftl

```diff
@@ -16,5 +16,7 @@ landing-all-value-prop-fix-exposures = We’ll help you fix your exposures
 landing-all-value-prop-fix-exposures-description = Our mission is to put control of your personal data back in your hands. We’ll help you resolve data breaches and keep your info private — and we’ll <privacy_link>respect your privacy</privacy_link> in the process.
 landing-all-value-prop-info-at-risk = What info could be at risk?
 landing-all-value-prop-info-at-risk-description = Data leaks are unfortunately part of our digital lives. Your passwords, contact details, financial information, and other personal info can be exposed, putting you at risk of identity theft.
-landing-all-value-prop-scanning-for-exposures-illustration-alt = Scanning for exposures progress bar
-landing-all-value-prop-leaked-password-illustration-alt = Leaked password example breach resolution step
+
+# Quote
+
+landing-all-quote = <data_breaches>Data breaches</data_breaches> happen every 11 minutes, exposing your private information — but don’t worry, we can help.
```

Comment:
It might be worth adding a comment on what the `<data_breaches>` tag does, so localisers understand what text it should apply to?